lib/Shortcut/ShortcutCommandKey.js

```diff
@@ -1,18 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ShortcutCommandKey = props => {
-  const ua = props.UAParser;
-
-  return (
-    <span className="shortcut__keyboard">
-      {ua.getOS().name === 'Mac OS' ? '⌘' : 'Ctrl'}
-    </span>
-  );
-};
+const ShortcutCommandKey = props => (
+  <span className="shortcut__keyboard">{props.mac ? '⌘' : 'Ctrl'}</span>
```

Comment:
This seems so simple now that I would argue that we should just use `props.children` and let the consuming app decide what content should be rendered.

It does make the component extremely simple but that's fine.